x/datapool/types/keys.go

```diff
@@ -51,3 +53,9 @@ func GetKeyPrefixDataValidateCertByRound(poolID, round uint64) []byte {
 func GetKeyPrefixDataValidateCert(poolID, round uint64, dataHash []byte) []byte {
 	return append(GetKeyPrefixDataValidateCertByRound(poolID, round), dataHash...)
 }
+
+func GetKeyPrefixNFTRedeemReceipt(poolID, round uint64, redeemer sdk.AccAddress) []byte {
```

Comment:
This key generated by using pool ID, round, and redeemer's address may cause conflicts.
If a buyer has multiple `DataPass`es and redeem them in the same round, the redeem receipt may have the same key. Thus, we may have to use token ID.

Reply:
Okay, it can be more better. If I applied the `tokenID`, we might need to modify our architecture docs later.